### ISSUE TITLE
artisan command passport isn't defined

### ISSUE THREAD

**Issue by fruckt** [Porto-SAP/Hello-API#63]
I have made installation manuali via git.

When I tried to run the command "php artisan passport:client --personal" from  DataBase Setup section I received error "[Symfony\Component\Console\Exception\CommandNotFoundException]  There are no commands defined in the "passport" namespace".

As I understend I have to install package Passport previuosly - but this isn't written in documentation for installation.

Regards
Yakov

**Comment by Mahmoudz** [comment]
@fruckt that's true I removed that from the Installation steps this morning. 

You will see some incorrect information in the current documentation because I was working on something and I paused it temporarily. Because I have to build a project really quickly for a client now.. I'll get back and continue all this after a month or two.

Basically I was replacing the existing Auth with Passport but then I found out that it will require a lot more time to finish all that. In addition to removing Dingo package and replacing it with Fractal Transporters.. And separating the Containers to be loaded by composer..

Some of the features are working but not documented and some have documentations but not implemented anymore.

I've added a note to the readme with an image to show that the project is currently under development and things will change. 

I appreciate that you are trying it, and I hope you will love it. 

After I make the first stable release I will be versioning the docs and the code so that we don't face similar problems in the future.

Best,
